web/experimental/styles.css

```diff
@@ -162,19 +164,74 @@ body {
 }
 
 .gutter.gutter-horizontal {
-  background-image: url('data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAUAAAAeCAYAAADkftS9AAAAIklEQVQoU2M4c+bMfxAGAgYYmwGrIIiDjrELjpo5aiZeMwF+yNnOs5KSvgAAAABJRU5ErkJggg==');
+  background-repeat: repeat-y;
```

Comment:
I think we may want to revisit the styling on the gutters in a subsequent PR. While the line-down-the-middle works pretty well on the existing embeds, it's causing some odd whitespace when the flutter display is up:

![image](https://user-images.githubusercontent.com/969662/59457919-3bf1ea00-8dce-11e9-8ca0-6987cde5a0b2.png)

Reply:
Yeah i was a little torn on this... we need the draggable area to have a certain width, and I wanted to go with Gayle's mockup for the dividers. 

Filed #1117 to track this